### PR TITLE
Susan edits for Thurs Oct 27

### PR DESCRIPTION
**Pull request by kassin**
Please incorporate these changes.

### CODE REVIEW

File: docs/api.rst [hunk 1]
```diff
@@ -1,5 +0,0 @@
-***
-API
-***
-
-This page documenta the underlying code base to aid MOSViz development.
```
Comment (pllim) [inline docs/api.rst]: Is this deletion intentional?

File: docs/readingindata.rst [hunk 2]
```diff
@@ -2,5 +2,58 @@
...
+Required Format and Header Information for Cut Out Images
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+********************************
+Example of Reading-in a Data Set
+********************************
+
+This example consists of galaxies at 0.2<z<1.2 which have spectra from the DEIMOS
+multi-object spectrograph on Keck and color cut-out images already created from 
+Hubble/ACS.  These galaxies are from the `DEEP2 Survey <http://adsabs.harvard.edu/abs/2013ApJS..208....5N>'_.
```
Comment (pllim) [inline docs/readingindata.rst]: There might be syntax errors in this doc. Should make sure Sphinx compiles properly without warning before merge.

**Comment by justincely** [comment]
I have some doc updates ready as well, once i get travis to build fully.  I think most won't have conflicts, but it may be easier to merge this request before merfing #41 so that susan doesn't have to resolve conflicts

**Comment by justincely** [comment]
I'm going to merge this in and check for syntax when I rebase against her changes

**Pull request merged**